### ISSUE TITLE
PMIx help text not found

### ISSUE THREAD

**Issue by wckzhang** [open-mpi/ompi#10096]
### What version of Open MPI are you using? (e.g., v3.0.5, v4.0.2, git branch name and hash, etc.)
OMPI master

### Describe how Open MPI was installed (e.g., from a source/distribution tarball, from a git clone, from an operating system distribution package, etc.)

git clone

```
--------------------------------------------------------------------------
Sorry!  You were supposed to get help about:
    version
But I couldn't open the help file:
    /fsx/ompi/install/share/pmix/help-schizo-ompi.txt: No such file or directory.  Sorry!
--------------------------------------------------------------------------
prterun: command line error (Out of resource)
```

I've been seeing these prints on recent debug runs. It doesn't seem to be causing any problems but some help text is missing.

**Comment by wzamazon** [comment]
I encountered the same problem. When I run `mpirun -h`, I got the following error message:

```
/fsx/ALinux2/PortaFiducia/libraries/openmpi/master/install/bin/mpirun -h
--------------------------------------------------------------------------
Sorry!  You were supposed to get help about:
    usage
But I couldn't open the help file:
    /fsx/ALinux2/PortaFiducia/libraries/openmpi/master/install/share/pmix/help-schizo-ompi.txt: No such file or directory.  Sorry!
--------------------------------------------------------------------------
```

**Comment by awlauria** [comment]
With the command line shuffling it isn't pointing to the right hostfile. libpmix is making the show-help call, so it expects it to be in a pmix subdir, when it actually is in a prte subdir (`install/share/prte`).

It is probably a simple fix, probably. prte and prun have the same issue. I'll post a patch for it this morning.

**Comment by awlauria** [comment]
Can you try these and add review/test:

https://github.com/openpmix/prrte/pull/1257
https://github.com/openpmix/openpmix/pull/2495

**Comment by awlauria** [comment]
v5.0.x is not effected, or shouldn't be. This came as a result of prrte using pmix utilities, which is not in the v5.0.0 series.

This should be fixed in the current prrte/pmix masters, just need to update the submodule pointer.

**Comment by david-edwards-arm** [comment]
I am seeing this issue on 5.0.0rc3 (built from tarball).

**Comment by awlauria** [comment]
Intersting, thanks. Will take a look.

**Comment by shijin-aws** [comment]
@awlauria I could verify that after checking out the commits in https://github.com/openpmix/prrte/pull/1257
and https://github.com/openpmix/openpmix/pull/2495 to prrte and openpmix, the error is fixed.

**Comment by awlauria** [comment]
Thanks @shijin-aws .

@david-edwards-arm are you seeing this with `mpirun --version / mpirun -V` ? 

I am able to reproduce that via the tarball, it seems the version option is broken in rc3. Other help options look ok to me.

**Comment by david-edwards-arm** [comment]
@awlauria yes I see the issue with `--version` but not `--help`; the issue affecting multiple executables `mpirun`, `prun`, `prte` etc.

**Comment by awlauria** [comment]
This should be fixed in master with #10136 fixed in master. Needs back porting to v5.0.0x

**Comment by awlauria** [comment]
This should be fixed in the v5.0.x branch via the following submodule pointer update: https://github.com/open-mpi/ompi/pull/10161

for main: https://github.com/open-mpi/ompi/pull/10165 (This PR fixes a variety of issues, but updating the submodule here will fix this issue on main).

**Comment by awlauria** [comment]
PR's merged. Closing. This (*I think*) made it into v5.0.0rc4, but if not will definitely be in v5.0.0rc5.
